.github/workflows/CI.yml

```diff
@@ -96,6 +96,7 @@ jobs:
       run:
         shell: bash
     env:
+      DAFFODIL_TEST_VSCODE_VERSION: 1.74.2
```

Comment:
This only pins in the GitHub CI, seems we should pin in runTests.ts and CI just tests with that version. Just like we don't want CI to randomly fail on stable updates, we don't want normal dev tests to fail on stable updates either.

Reply:
Why exactly don't we want normal dev tests to fail on stable updates?  I see this as something we should welcome and not avoid.  IMHO, we ought to be actively testing against the latest stable release.  CI can pin it against some "known good" so we don't regress against a particular historical version, which I believe is sufficient.  This project in general has not been great at keeping up to date with dependency versions and pinning the version inside the code will lead to yet another one of these versions that don't get updated with the times.  The VS Code version is under the end user's control, which is likely to be something more recent than what we'd be testing against.

Reply:
I should be able to test commit XYZ now and in some time in the future and should be guaranteed to should pass. Likewise, I should be able to download an old release and be guaranteed its tests pass no matter when I choose to run it and what stable happens to be at that time.

If we don't version pin and stable makes a breaking change, then that old commit or release that used to pass might fail and there's nothing you can do about it since we don't know what stable was at that point in time.

> IMHO, we ought to be actively testing against the latest stable release

100% agreed, and I would even claim we should be testing against beta/unstable/whatever if VS code has those so we catch any issues before they hit stable. But we should be doing so with automated builds/tests, and automated PRs that update the version to latest depenencies. We shouldn't need to rely on devs to run tests periodically enough to catch issues when a computer can do it on a much more frequent and reliable basis.

And If a dev wants to always test with the latest and greatest and have the chance for failure and catch any issues earlier, then they can set the environment variable to stable, and can even do so globally.

> This project in general has not been great at keeping up to date with dependency versions

Then this needs to be improved. There are now at least three PMC members that focus primarily on Daffodil VS Code (as compared to Daffodil itself, I definitely understand there are other non Daffodil related priorities) and can review and merge those changes. If the number is too overwhelming then we should make changes to only submit PRs for the most critical dependencies or only major releases or something, and ignore all the others.